README.md

```diff
@@ -37,14 +37,31 @@ lein run bundle.tar.gz --input '{"user": {"roles": ["admin"]}}'
 Note that the above constitutes a simple flow for development and testing only. Production artifacts, or a fixed API for
 integrations, will be here at a later point in time.
 
+### Clojure
+
+Run `lein repl` to launch a REPL in the `jarl.core` namespace:
```

Comment:
Don't you need to `cd` into the `core` dir first?

Reply:
Yeah that's true, I'll add that. I also realized we have a .clj file to run this, so will add that as well.